client/README.md

```diff
@@ -37,6 +37,10 @@ Instead, it will copy all the configuration files and the transitive dependencie
 
 You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
 
+### `npm install @reach/menu-button`
```

Comment:
I think that instead of requiring the user to run an npm install command, it would probably be better to just add @reach/menu-button to the dependencies section of the client/package.json file. That way, the dependency is automatically installed when you run npm install instead of relying on someone to do it themselves.